frontend/src/services/utils.ts

```diff
@@ -46,3 +48,21 @@ export const formValidationSchema = (isAbstractRoleSelected: boolean) => {
             .nullable(),
     });
 };
+
+export const isNewAccessId = (
+    accessesData: number,
+    newAccessId: number | null
+) => {
+    return accessesData === newAccessId;
+};
+
+export const resetNewTag = () => {
+    router.push({
+        name: routeItems.dashboard.name,
+        params: {
+            newUserAccessIds: '',
+            newAppAdminId: '',
+            newDelegatedAdminIds: '',
+        },
+    });
+};
```

Comment:
This seems to be very specific only for dashboard route, probably shouldn't be in /servics/utils.ts